src/app/options.cpp

```diff
@@ -247,6 +252,12 @@ namespace
     constexpr const StringOption CONFIGURATION_OPTION = {"configuration"};
     constexpr const BoolOption PORTABLE_OPTION = {"portable"};
     constexpr const BoolOption RELATIVE_FASTRESUME = {"relative-fastresume"};
+    constexpr const StringOption SAVE_PATH_OPTION = {"path"};
+    constexpr const StringOption STATUS_OPTION = {"status"};
```

Comment:
@evsh It's mainly because, without having an `Option`-derived class that can return a tri-state bool, the mechanism where options can be specified using environment variables no longer works.

When initializing a value in the constructor of `QBtCommandLineParameters` you need to use something like: `addPaused(PAUSED_OPTION.value(env))`

This will initialize the variable `addPaused` to either true or false. So if we don't get a value from the corresponding environment variable, `addPaused` is initialized to false rather than to undefined. And now we don't know whether the user actually specified false in the environment variable or didn't specify anything at all. Using `TriStateBoolOption` fixes this.

On top of that, and perhaps even more importantly, I think having a separate class called `TriStateBoolOption` is *conceptually* correct as it provides a different function (namely the ability to have an undefined state) in comparison to `BoolOption`. Better to make that explicit in the definition of an option than to have its behavior not come from its definition but its implementation details. You don't want people to have to check the code carefully to know that `add-paused` behaves differently than every other `BoolOption`.